Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "log-broker"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0"
+bb8-redis = "0.13"
+futures-util = "0.3"
+lazy_static = "1.4"
+tokio = { version = "1", features = ["macros", "rt-multi-thread"] }
```

Comment:
I think some of the dependencies like Tokio can be moved to dev_dependecies but things such as bb8-redis and futures-util might need to be kept in dependencies since they are needed for the functions to run and not only just the unit tests.

Reply:
Actually I think the only thing that could be moved to dev_dependencies is tokio since bb8-redis and lazy_static is needed for the lib to run however, futures-util can be removed.

Reply:
Done, moved Tokio to dev-dependencies